x-pack/plugins/canvas/public/components/workpad_config/index.js

```diff
@@ -21,12 +21,14 @@ const mapStateToProps = state => {
       width: get(workpad, 'width'),
       height: get(workpad, 'height'),
     },
+    style: get(workpad, 'style') || '.canvasPage {\n\n}',
```

Comment:
`.canvasPage {\n\n}` also appears in the component. Break that out into a constant called `defaultWorkpadCSS`